### ISSUE TITLE
about training dataset

### ISSUE THREAD

**Issue by 99creater99** [gentlefress/RoboGhost#4]
I'm trying to reproduce it, but I've run into a bit of a problem regarding the Stage_2 training dataset. I could only find 472 entries in the publicly available dataset, but from the paper, it seems that there should be more entries that can be used for reinforcement learning after filtering, and there should be a split between the test set and training set. Am I misunderstanding something? I’d really appreciate your reply, thank you so much!
我正在尝试复现，但是遇到了一点问题，关于Stage_2的训练数据集，公开的数据集只找到472条数据，但是从论文中看筛选后可用于强化学习训练的数据条目应该更多，并且会有测试集和训练集的划分，请问是否是我的理解问题？期望能得到您的答复，万分感谢！
